### DISCUSSION TITLE
Aligning style guides with DevRel

### DISCUSSION THREAD

**Discussion by billy-the-fish** [AgoraDoc/agora-style-guide#28] [Ideas]
A lot of content is duplicated between DevRel and the Documentation. In the long term is is a good idea to move a lot of DevRel content to the docs. This also leaves DevRel free to write more evangelistic blog posts and not the nuts and bolts posts they are doing now. The easiest way to do this is if we write in the same way. DevRel have agreed to adopt our single page doc template so we can easily migrate content. This will be even easier if we use a common style guide. 

DevRel have a simple style guide that mostly agrees with the doc one but differs on a few points. The doc style guide is mostly the same as the Google Developer Style guide. What I propose is that in the spirit of using global norms and keeping it simple, we adopt the Google Style Guide as the base and ONLY add things specific to Agora in our joined style guide. 

In the future docs-as-code world, this will also make it easier for developers to write API docs and procedures if they so choose.

**Comment by jiewang2** [comment]
Great suggestions! 
Could you point me to the style guide of DevRel? Maybe I can first simply the current style guide and compare it with the one from DevRel, then we can discuss the differences.

**Reply by billy-the-fish** [reply]
Happy reading: https://github.com/AgoraIO-Community/ContentContributorGuides/blob/master/content-contributor-program-writing-guidelines.pdf

**Reply by jiewang2** [reply]
Thanks. I went through this and here are some findings for discussion:

- Things that differ from our style guide:
  - Use title case for headings and subheadings. We use sentence case.
  - Word list. We need to combine this with the one in our style guide.
  - Include images and videos. We do not have this in the current style guide, but recently we had a discussion about screenshots. We would like to keep screenshots to the minimum. @HeatherWYL 
- Things that are not in our style guide:
  - Stay relevant to the title
  - Use bulleted lists
  - Never offer legal advice
  - Keep SEO principles in mind
  - Use copyrights and trademarks carefully
